notebooks/infer_predictions.ipynb

```diff
@@ -0,0 +1,180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Run inference on pre-trained model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "import torch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from grace.io.image_dataset import ImageGraphDataset\n",
+    "from grace.models.feature_extractor import FeatureExtractor\n",
+    "from grace.evaluation.visualisation import plot_simple_graph\n",
+    "from grace.evaluation.inference import GraphLabelPredictor\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read some real grace-annotated data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "extractor_filename = \"/Users/kulicna/Desktop/classifier/extractor/resnet152.pt\"\n",
```

Comment:
If we want people to be able to run this easily maybe we need to add the line to download the resnet before or add a comment to point to where this is.

Reply:
Great point! I included the instructions on how to download the ResNet locally in the `training` subfolder README:
https://github.com/alan-turing-institute/grace/blob/488f60bf6822106129c1de1c83b244c051dfc827/grace/training/README.md?plain=1#L67-L78
Would you like to see it added in the notebook, too?